releasenotes/notes/deprecate-process_config.enabled-89e9ab7b0193cc04.yaml

```diff
@@ -0,0 +1,7 @@
+---
+features:
+  - |
+    Added `process_config.process_collection.enabled` and `process_config.container_collection.enabled` settings.
+    Use these new settings instead of `process_config.enabled` to control continer and process collection in the process agent.
+deprecations:
+  - Deprecated `process_config.enabled`.
```

Comment:
Could you please double check how deprecations appear in the change log - if they are in a separate section, you may want to duplicate some of the content under `features` to make it clear we are replacing this setting with different configuration.

Reply:
The changelog is rendered [here](https://github.com/DataDog/datadog-agent/blob/main/CHANGELOG.rst). Deprecations show in their own separate section, so I will make sure to duplicate some of the content